### PR TITLE
Writing tutorial for Voxels

### PR DESCRIPTION
**Pull request by Zairka22**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,53 @@
-# voxels-tutorial
\ No newline at end of file
+# voxels-tutorial
```
Comment (PacificUnific) [inline README.md]: Make a "good" title

File: README.md [hunk 2]
```diff
@@ -1 +1,71 @@
...
+* Click on Download
+    * Save file on PC
+* Click on To Voxels button
+## 3. Connecting with MetaMask
+![Image](./media/2.png)
+* Click on Sign In
+* Click on MetaMask
+	* Click on Next
+	* Click on Connect
+	* Click on Sign
```
Comment (PacificUnific) [inline README.md]: Leave the step "Sign In Voxels" only, because user can auth with other wallet

File: README.md [hunk 3]
```diff
@@ -1 +1,71 @@
...
+## 4. Registration and signing
+![Image](./media/3.png)
+* Enter Name
+* Click on Register name
+* On MetaMask click on Sign
```
Comment (PacificUnific) [inline README.md]: Note: it is not necessary

File: README.md [hunk 4]
```diff
@@ -1 +1,71 @@
...
+* Click on MetaMask
+	* Click on Next
+	* Click on Connect
+	* Click on Sign
+## 4. Registration and signing
+![Image](./media/3.png)
+* Enter Name
+* Click on Register name
+* On MetaMask click on Sign 
+## 5. Creating a space
```
Comment (PacificUnific) [inline README.md]: Point: all described below relates to parcel deploy also

File: README.md [hunk 5]
```diff
@@ -1 +1,71 @@
...
+* Click on Open
+* Click on OK
+## 17.
+![Image](./media/12.png)
+* Enter name
+* Enter description
+* Upload picture
+* Click on Save  
+<u>EXPECTED STATE</u>: Voxel ad space displayed in list
+You can start campaigns. Available for voxel - picture creatives, video creatives, YouTube stream, twitch stream.
```
Comment (PacificUnific) [inline README.md]: picture -> image
voxel -> Voxels

File: README.md (hunk 1 above)
Comment (Zairka22) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (Zairka22) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (Zairka22) [reply README.md]: ok

File: README.md (hunk 5 above)
Comment (Zairka22) [reply README.md]: done

File: README.md [hunk 6]
```diff
@@ -1 +1,79 @@
-# voxels-tutorial
\ No newline at end of file
+# MetaAds Voxel's Scene
+
+## Introduction
+
+This document is an instruction on how to create a scene for Voxels.
```
Comment (PacificUnific) [inline README.md]: add hyperlink to Voxels

File: README.md [hunk 7]
```diff
@@ -1 +1,79 @@
...
+
+## 1. Opening the MetaAds website
+
+* Connect
```
Comment (PacificUnific) [inline README.md:9]: add hyperlink to MetaAds site

File: README.md [hunk 8]
```diff
@@ -1 +1,79 @@
...
+
+![Image](./media/14.png)
+![Image](./media/12.png)
+![Image](./media/13.png)
+
+* Click on Add Ad Space
+* Choose Voxels metaverse
+* Click on Download
+    * Save file on PC
+* Click on To Voxels button
```
Comment (PacificUnific) [inline README.md]: text and picture, one by one

File: README.md [hunk 9]
```diff
@@ -1 +1,79 @@
...
+![Image](./media/3.png)
+![Image](./media/4.png) 
+
+* Click on Spaces
+* Click on Create a space
+* Enter Name
+* Choose a size of scene
+* Click on Create  
+
+## 7. Adding the Meta Ads asset
```
Comment (PacificUnific) [inline README.md]: wrong number

File: README.md [hunk 10]
```diff
@@ -1 +1,79 @@
...
+## 4. Creating a space
+
+![Image](./media/3.png)
+![Image](./media/4.png) 
+
+* Click on Spaces
+* Click on Create a space
+* Enter Name
+* Choose a size of scene
+* Click on Create
```
Comment (PacificUnific) [inline README.md]: text and picture after it

File: README.md [hunk 11]
```diff
@@ -1 +1,79 @@
...
+
+* Click on Add
+* Click on Browse Asset Library
+* Click on Meta Ads asset (if not listed, enter the name in the search)
+* Click on Add to Parcel
+* Position the object on the land 
+* Click on Advanced 
+* Insert downloaded script to Script field 
+
+> If the red circle next to the Expand button doesn't turn green, reload the page(if the circle turns green, close this form).
```
Comment (PacificUnific) [inline README.md]: space before a parenthesis

File: README.md [hunk 12]
```diff
@@ -1 +1,79 @@
...
+
+## 8. Completing the creation of the scene
```
Comment (PacificUnific) [inline README.md]: wrong number

File: README.md (hunk 11 above)
Comment (PacificUnific) [reply README.md]: if the circle **has turned** green, close this form

File: README.md [hunk 13]
```diff
@@ -1 +1,79 @@
...
+![Image](./media/6.png)
+![Image](./media/7.png)
+![Image](./media/8.png)
+
+* Click on Add
+* Click on Browse Asset Library
+* Click on Meta Ads asset (if not listed, enter the name in the search)
+* Click on Add to Parcel
+* Position the object on the land 
+* Click on Advanced
```
Comment (PacificUnific) [inline README.md]: Click on Advanced \<point a feature name\>

File: README.md (hunk 6 above)
Comment (Zairka22) [reply README.md]: done

File: README.md (hunk 8 above)
Comment (Zairka22) [reply README.md]: done

File: README.md (hunk 9 above)
Comment (Zairka22) [reply README.md]: done

File: README.md (hunk 10 above)
Comment (Zairka22) [reply README.md]: done

File: README.md (hunk 11 above)
Comment (Zairka22) [reply README.md]: done

File: README.md (hunk 12 above)
Comment (Zairka22) [reply README.md]: done

**Pull request merged**